docs/setup-cluster/deploy-cluster/slurm/slurm-requirements.rst

```diff
@@ -202,6 +202,27 @@ interacts with Slurm, we recommend the following steps:
 Determined should function with your existing PBS configuration. To optimize how Determined
 interacts with PBS, we recommend the following steps:
 
+-  Enable PBS to store job history.
+
+   In order for the HPC launcher to detect successful job completion, PBS must be configured such
+   that ``job_history_enable = True``. If this setting is not configured properly, then the HPC
+   launcher will not be able to resolve the status/information of a job after it finishes.
+
+   PBS administrators can use the following commands to set/verify the value of
```

Comment:
Instead of "set/verify" maybe say "set and verify"?

Reply:
I was almost wondering about just dropping the verify.   If they set it and don't get an error it is set :-)